afs_ioc_migration/hook_template.txt

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
As discussions today made clear, if there's a friction point in this whole process it will be uncovered here.  

I wish we had some more verbose documentation for this we could point to.  (Despite the workflow you describe being stock standard for us now)